rpms/build.sh

```diff
@@ -53,6 +53,7 @@ fi
 build_dir=/build_wazuh
 rpm_build_dir=${build_dir}/rpmbuild
 file_name="wazuh-${build_target}-${wazuh_version}-${package_release}"
+dbg_file="wazuh-${build_target}-debuginfo-${wazuh_version}-${package_release}"
```

Comment:
Missing architecture type in the name

Reply:
This is a template to retrieve packages and put them into the output folder. The template matches the standard package syntax